### PR TITLE
Blessings reworked AGAIN

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
Makes some adjustments to our big contenders when it comes to abnormality blessings.

- Puss in boots will no longer be passive if breaching due to a Qlipthoth meltdown, and will hunt down the blessed player and remove blessings as usual during a hostile breach.
- Puss in boots is now an actual threat during a hostile breach, dealing a whopping 125 red damage on a successful execution attack. Of course, this requires you to stay in line of sight for about 4 seconds...
- The Knight of Despair will only provide a blessing on a successful attachment work.
- In addition, the Knight of Despair will remove existing blessings and breach if its Qlipthoth counter reaches 0 three times for any reason. There is a visual indicator for this, and there are no normal means to reduce Qlipthoth counter. Only things that always reduce counter such as crimson clowns and failed meltdowns will reduce the abnormality's counter.

## Why It's Good For The Game
Makes the game less broke!

## Changelog
:cl:
balance: rebalanced blessings
/:cl:

### CODE REVIEW

**Comment by CrabbytheCrab** [comment]
ro mains in shambles

**Comment by LanceSmites328** [comment]
Why the switch to Red from Pale for Puss? 50 Pale was a lot on basically *anyone*, regardless of level. Changing it to Red primarily serves killing lower-stat and less-equipped people.

**Comment by Coxswain-Navigator** [comment]
> Why the switch to Red from Pale for Puss? 50 Pale was a lot on basically _anyone_, regardless of level. Changing it to Red primarily serves killing lower-stat and less-equipped people.

This is true, it always checked for red defense though. And besides that, I think killing lower-stat and less-equipped people makes the prospect of losing the blessing that much scarier.

**Comment by LanceSmites328** [comment]
> > Why the switch to Red from Pale for Puss? 50 Pale was a lot on basically _anyone_, regardless of level. Changing it to Red primarily serves killing lower-stat and less-equipped people.
> 
> This is true, it always checked for red defense though. And besides that, I think killing lower-stat and less-equipped people makes the prospect of losing the blessing that much scarier.

I don't? Lower-stat and less equipped people are, to put it bluntly, less important. No one's gonna cry if Clerk Lacy dies compared to LV.5 Agent McBadass. The difference here is overall a difference in threat to the round as a whole versus individual persons.

**Comment by Coxswain-Navigator** [comment]
> > > Why the switch to Red from Pale for Puss? 50 Pale was a lot on basically _anyone_, regardless of level. Changing it to Red primarily serves killing lower-stat and less-equipped people.
> > 
> > 
> > This is true, it always checked for red defense though. And besides that, I think killing lower-stat and less-equipped people makes the prospect of losing the blessing that much scarier.
> 
> I don't? Lower-stat and less equipped people are, to put it bluntly, less important. No one's gonna cry if Clerk Lacy dies compared to LV.5 Agent McBadass. The difference here is overall a difference in threat to the round as a whole versus individual persons.

Alright, i'll change it back.

File: code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm [hunk 1]
```diff
@@ -44,6 +39,26 @@
...
+			playsound(get_turf(src), 'sound/abnormalities/despairknight/attack.ogg', 50, 0, 4)
+			datum_reference.qliphoth_change(1)
+		else
+			cut_overlays()
+			if(blessed_human)
+				BlessedDeath()
+			else
+				BreachEffect()
+			return
+	swords += 1
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm:61]: Seems a bit odd; Is there any reason to *kill* the blessed? Don't turn KoD into ML copy.

File: code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm [hunk 2]
```diff
@@ -6,25 +6,20 @@
...
-
 	maxHealth = 2000
 	health = 2000
 	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.2, WHITE_DAMAGE = 1.0, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 0.5)
 	stat_attack = HARD_CRIT
-
 	del_on_death = FALSE
 	deathsound = 'sound/abnormalities/despairknight/dead.ogg'
-
-	move_to_delay = 4
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm:25]: ?

Comment (Coxswain-Navigator) [reply code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm:25]: This is an oversight, good catch.

File: code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm (hunk 1 above)
Comment (Coxswain-Navigator) [reply code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm:61]: This proc does not actually kill the blessed.

**Review by EgorDinamit — APPROVED** [review]

**Pull request merged**